src/main/java/cn/nukkit/Player.java

```diff
@@ -3307,6 +3307,7 @@ public void onCompletion(Server server) {
 
                                         this.inventory.setItemInHand(item);
                                     }
+                                    //TODO : Trigger useOn in creative mode.
```

Comment:
I might do this TODO if nobody does it 😊

Reply:
This is actually not needed, `Item.useOn` is to damage the item when it is used, since the player is in creative mode the item should not be damaged. The `Entity.onInteract()` method is still called by players in any game mode correctly, so I removed the TODO.

Reply:
I've noticed that after i've sent the second commit.